base/_typography.scss

```diff
@@ -238,11 +232,172 @@ table {
 	}
 
 	// zebra striping pattern
-	&.striped {
-		tbody {
-			tr:nth-of-type(odd) {
-				background-color: $table-border-color;
-			}
+	tbody {
+
+		tr:nth-of-type(odd) {
+			background-color: $table-border-color;
+		}
+	}
+}
+
+// Inline typography elements
+strong,
+b,
+em,
+i,
+cite,
+small,
+sup,
+sup { letter-spacing: .03125em; }
+
+sub,
+sup {
+	font-size: 75%;
+	line-height: 1;
+	position: relative;
+}
+
+sup { top: -0.5em; }
+
+sub { bottom: -0.25em; }
+
+abbr[title],
+cite,
+dfn[title] { border-bottom: 1px dotted palette(brand); }
+
+abbr[title],
+dfn[title] { cursor: help; }
+
+abbr[title] {
+	font-size: 75%;
+	font-weight: 700;
+	letter-spacing: 0.125em;
+	text-transform: uppercase;
+}
+
+b,
+strong { font-weight: 700; }
```

Comment:
I'm not sure that bold always translates to the same numerical weight for every browser. Maybe this should be a rule that we agree on… always use the numeric weight esp. when using Google fonts.